articles/azure-app-configuration/enable-dynamic-configuration-dotnet-core-push-refresh.md

```diff
@@ -80,11 +80,16 @@ Once the resources are created, add the following environment variables. These w
 > [!NOTE]
 > When subscribing for configuration changes, one or more filters can be used to reduce the number of events sent to your application. These can be configured either as [Event Grid subscription filters](../event-grid/event-filtering.md) or [Service Bus subscription filters](../service-bus-messaging/topic-filters.md). For example, a subscription filter can be used to only subscribe to events for changes in a key that starts with a specific string.
 
-## Register event handler to reload data from App Configuration
+## Register event handler to reload data from App Configuration using a sync-token
 
+> [!NOTE]
+> Support for using synchronization tokens in push refresh workflow is available in .NET configuration provider version 5.0.0-preview (or later). 
```

Comment:
Can we update the prereq section with the following instead?

- Quickstart: [Create a .NET Core app with App Configuration](./quickstart-dotnet-core-app.md)
- Nuget package `Microsoft.Extensions.Configuration.AzureAppConfiguration` version 5.0.0 or later

(I left out -preview. When this doc goes out, we should have 5.0.0 released.)